luigi/contrib/mongodb.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Big Datext Inc
```

Comment:
I took that from [Twitter](https://github.com/spotify/luigi/blob/2dcfeaa5773d54582ad9d8546394685636ce4783/test/contrib/bigquery_test.py) contribution, but I can change that, we do not mind at all.

I do not know the policy about that, anyway it is open source :)